python/ray/util/annotations.py

```diff
@@ -121,13 +126,70 @@ def Deprecated(*args, **kwargs):
     def inner(obj):
         if not obj.__doc__:
             obj.__doc__ = ""
-        obj.__doc__ += f"{message}"
+
+        obj.__doc__ = obj.__doc__.rstrip()
+
+        indent = _get_indent(obj.__doc__)
+        obj.__doc__ += "\n\n"
+        obj.__doc__ += f"{' ' * indent}.. warning::\n"
```

Comment:
Can you also please update the other annotations in this file?

Reply:
Done